sort/insertion.py

```diff
@@ -1,2 +1,16 @@
 def sort(array: list[int]) -> list[int]:
+    """
+    Insertion sort algorithm
+
+    :param array:
+    :return: sorted array
+    """
+    for i in range(1, len(array)):
+        key = array[i]
+        j = i - 1
+        while j >= 0 and key < array[j]:
+            array[j + 1] = array[j]
+            j -= 1
```

Comment:
Tu modifies le tableau à chaque itération. On peut le modifier une seule fois, après avoir trouvé le bon index.